src/formats/newick.rs

```diff
@@ -44,7 +44,7 @@ where
             out_buf.push_back(NewickToken::Start);
         } else {
             out_buf.push_back(NewickToken::End);
-            let mut name: String = format!("{}", node);
+            let mut name: String = node.to_string().split_whitespace().collect();
```

Comment:
@Keats I'm not sure if this is still necessary, it was in the branch prior to merging with upstream

Reply:
reverted